src/app/pages/details/instructor-resource-tab/instructor-resource-tab.jsx

```diff
@@ -0,0 +1,149 @@
+import React, {useState, useEffect} from 'react';
+import FeaturedResources from './featured-resources/featured-resources.jsx';
+import {instructorResourceBoxPermissions} from '../resource-box/resource-box';
+import ResourceBoxes from '../resource-box/resource-boxes.jsx';
+import partnerFeaturePromise, {tooltipText} from '~/models/salesforce-partners';
+import shuffle from 'lodash/shuffle';
+import Partners from './partners/partners.jsx';
```

Comment:
Tutor uses `babel-plugin-transform-imports` so we can import lodash like `import { shuffle } from 'lodash';` 
https://github.com/openstax/tutor-js/blob/master/babel.config.js#L23-L26